crates/uv/src/commands/tool/upgrade.rs

```diff
@@ -101,11 +136,26 @@ pub(crate) async fn upgrade(
         writeln!(printer.stderr(), "Nothing to upgrade")?;
     }
 
+    if let (true, Some(python)) = (did_upgrade, python) {
+        writeln!(
+            printer.stderr(),
+            "Upgraded build environment for {} to Python {}",
+            if names.len() == 1 {
```

Comment:
I would use `if let [name] = names.as_slice()` here so that you can avoid the `unwrap` below.

Reply:
Done